rulesets/resources/301ruleset.rb

```diff
@@ -612,32 +612,52 @@ def self.set_enclosure_walls_reference(orig_bldg, new_bldg)
     ufactor = get_reference_wall_ufactor()
 
     ext_thermal_bndry_walls = orig_bldg.walls.select { |wall| wall.is_exterior_thermal_boundary }
-    sum_gross_area = ext_thermal_bndry_walls.map { |wall| wall.area }.sum(0)
+    sum_ext_wall_gross_area = ext_thermal_bndry_walls.map { |wall| wall.area }.sum(0)
+    other_walls = orig_bldg.walls - ext_thermal_bndry_walls
 
     solar_absorptance = 0.75
     emittance = 0.90
 
     # Create insulated walls for exterior thermal boundary surface.
     # Area is equally distributed to each direction to be able to accommodate windows,
     # which are also equally distributed.
-    if sum_gross_area > 0.1
+    if sum_ext_wall_gross_area > 0.1
       new_bldg.walls.add(id: 'WallArea',
                          exterior_adjacent_to: HPXML::LocationOutside,
                          interior_adjacent_to: HPXML::LocationConditionedSpace,
                          wall_type: HPXML::WallTypeWoodStud,
-                         area: sum_gross_area,
+                         area: sum_ext_wall_gross_area,
                          azimuth: nil,
                          solar_absorptance: solar_absorptance,
                          emittance: emittance,
                          insulation_assembly_r_value: (1.0 / ufactor).round(3))
     end
 
+    # Create walls for Above-grade walls separating Conditioned Space Volume
+    # from Unconditioned Space Volume, Unrated Heated Space, Multifamily Buffer Boundary,
+    # or Non-Freezing Space.
+    if Constants.ERIVersions.index(@eri_version) >= Constants.ERIVersions.index('2022')
+      common_space_walls = orig_bldg.walls.select { |wall| wall.is_conditioned_and_adjacent_to_multifamily_common_space }
```

Comment:
@yzhou601 This doesn't look quite right to me because it only captures the MF common spaces, but the language also includes "Above-grade walls separating Conditioned Space Volume from Unconditioned Space Volume". So I think it could simply be:
```suggestion
      common_space_walls = orig_bldg.walls.select { |wall| wall.is_thermal_boundary }
```

And then we should probably update all the "common_space" references to something more generic. Maybe "buffer_space"?

Reply:
There may be something I missed, are you talking about the PDS03 Table 4.2.2(1)? The language there is "Above-grade walls separating Conditioned Space Volume from Unrated Heated Space, Multifamily Buffer Boundary, or Non-Freezing Space" where the U factors are changed to be 0.292 or 0.089 based on IECC climate zones.

Reply:
I was reading the comment in your code. Hmm, looks like we need to track this down...

Reply:
Ah I see, I think the comment is not correct. I should have removed the unconditioned volume, the language "Above-grade walls separating Conditioned Space Volume from Unrated Heated Space, Multifamily Buffer Boundary, or Non-Freezing Space" is copied from the standard.

Reply:
I see what happened. PDS 01 had the language you referenced, but then it got changed (fixed) in PDS 02. Here's the change in PDS 02:

![image](https://github.com/NREL/OpenStudio-ERI/assets/5861765/d4f46f4b-dff3-4b7a-98fd-a8c575f827a4)

So yes, what you have looks good.

Reply:
And then PDS 03 changed it one more time (the U-factors only):

![image](https://github.com/NREL/OpenStudio-ERI/assets/5861765/edd33475-3d63-4323-aa8b-78093dc533b2)

Reply:
Thanks for tracking this down! Yep I guess what I did is that I directly copied the languages in PDS03 and the struck out was incorrectly kept by copy-paste, the reference links on trello cards are extremely helpful on this kind of frequently changed items!

Reply:
Ah, that makes sense. That's funny.